### PR TITLE
style: fixes panic bug and code style

### PR DESCRIPTION
**Pull request by sunpe**
1. fix `panic` bug for `Processor` counter if value < 0
2. fix `panic` bug for `adapter.go` when http error is not null
3. for{ select {} }  --> for range {}

### CODE REVIEW

**Review by huolibo — APPROVED** [review]
LGTM

File: process/handle.go [hunk 1]
```diff
@@ -81,8 +80,12 @@ func (p *Processor) Collect(metrics chan<- prometheus.Metric) {
 					if value.Value == nil {
 						continue
 					}
+					v := i2float(value.Value)
+					if v < 0 {
+						continue
```
Comment (huskar-t) [inline process/handle.go:85]: 不打印个日志吗

**Review by huskar-t — APPROVED** [review]
LGTM

File: process/handle.go (hunk 1 above)
Comment (sunpe) [reply process/handle.go:87]: 已修改

**Pull request merged**